scripts/sexbound/plugins/climax/climax.lua

```diff
@@ -130,6 +130,10 @@ function Sexbound.Actor.Climax:onUpdateSexState(dt)
 
     local multiplier = actor:getPosition():getAnimationState("sexState"):getClimaxMultiplier(actorNumber) or 1
     local increase = util.randomInRange(self:getDefaultIncrease()) * multiplier * dt
+    -- increase point gain if this is a defeat node to allow for climaxing during the limited sexnode duration
+    if self:getParent():getParent():getContainsDefeated() then
+        increase = increase * 2.1
```

Comment:
Should potentially be made a config option, to be changed alongside how long the node stays.

Reply:
That's a good idea, I'll look into implementing that.